### ISSUE TITLE
question about dataset

### ISSUE THREAD

**Issue by 2034492239** [waymo-research/waymo-open-dataset#694]
I have a few questions:

Perception dataset has 2,030 scenes, and the Motion dataset has 103,354 scenes. However, in the latest data files, the perception dataset only has a little over 1,000, and the motion dataset is the same. Do each TFRecord file contain multiple scenes, or is it from the historical dataset?

The recent Motion dataset added Lidar data and driveways. So, is there any way for me to know if the scenes are from urban driving or highway driving?

**Comment by scott-ettinger** [comment]
Yes, the motion dataset contains multiple examples (scenes) per record.  We do not currently provide a way to determine the type of scene or location for the motion data.  We do provide the lane types one of which is [TYPE_FREEWAY](https://github.com/waymo-research/waymo-open-dataset/blob/656f759070a7b1356f9f0403b17cd85323e0626c/src/waymo_open_dataset/protos/map.proto#L149) in the map data.

The perception dataset contains a single scene per file, but multiple records for each time step.

**Comment by BenQLange** [comment]
Clarification on the above response, should we expect 2,030 independent segments, i.e. 2,030 20-second segments with sensor measurements at 10 Hz?

I've loaded a v2 version of the dataset with lidar measurements only. I am seeing over 1000 parquet files with approx. 200 Range images (20 seconds x 10Hz) for top lidar each. That would be then 1000 segments. Am I misunderstanding something? Thanks!
